### PR TITLE
ROC-5567 settlement dashboard tests

### PR DESCRIPTION
**Pull request by andrewwa-kainos**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-5567

### Change description
Add content route tests for settlement states on the dashboard and claim-status pages

### Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: src/main/app/claims/models/claim.ts [hunk 1]
```diff
@@ -91,9 +91,7 @@ export class Claim {
     return !this.countyCourtJudgmentRequestedAt
       && (this.admissionPayImmediatelyPastPaymentDate
         || this.hasDefendantNotSignedSettlementAgreementInTime()
-        || (!this.respondedAt && isPastDeadline(MomentFactory.currentDateTime(), this.responseDeadline)
-        || this.isSettlementAgreementRejected()
```
Comment (sabahirfan) [inline src/main/app/claims/models/claim.ts]: The PR says only about adding tests.
This condition (`this.isSettlementAgreementRejected()`) is removed, @andrewwa-kainos I hope it's not accidental.

Comment (andrewwa-kainos) [reply src/main/app/claims/models/claim.ts]: There was an understanding that any errors found would be fixed (hence some of the other changes). This change was more about cleaning up redundancy though.

Comment (sabahirfan) [reply src/main/app/claims/models/claim.ts]: Thanks

File: src/main/features/dashboard/views/macro/caseHistory.njk [hunk 2]
```diff
@@ -159,12 +159,12 @@
       {{ offerSubmittedForClaimantClaimDetails(claim) }}
     {% elif state.status === ClaimStatus.PART_ADMIT_PAY_IMMEDIATELY %}
       {{ partAdmittedPayImmediatelyClaimantClaimDetails(claim) }}
+    {% elif state.status === ClaimStatus.OFFER_SETTLEMENT_REACHED %}
```
Comment (simad) [inline src/main/features/dashboard/views/macro/caseHistory.njk]: I'm so glad we're refactoring this `mess` :D

**Pull request merged**